docs/essentials/controls.md

```diff
@@ -147,35 +151,20 @@ As they can be complex cases:
 
 <!-- prettier-ignore-end -->
 
-Use customized options: 
-
-<!-- <!-- prettier-ignore-start -->
-
-<CodeSnippets
-  paths={[
-    'common/component-story-custom-args-complex-object.js.mdx',
-  ]}
-/>
-
-<!-- prettier-ignore-end -->
-
-Or even with certain types of elements, such as icons:
+Unless you need the flexibility of a function, an easier way to map primitives to complex values before rendering is to define a `mapping`. Additionally, you can specify `control.labels` to configure custom labels for your checkbox, radio or select input.
```

Comment:
what happens if you need the flexibility of a function? we're probably better off not mentioning this at all?

Reply:
Should we omit the example with the Template altogether then? I would be totally fine with that.

Reply:
I mis-read this paragraph the first time around and thought you were referring to the mapping function which we decided not to implement. I'm OK with this as-is.